src/styles/_catalog.scss

```diff
@@ -1,22 +1,19 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
```

Comment:
check task once again, your doesn't work properly:
There should be four columns starting at 1024px
There should be three columns starting at 768px
There should be two columns starting at 488px